lib/worker/src/db/publications.rs

```diff
@@ -65,21 +68,16 @@ pub async fn pub_cids(
     rel: String,
     limit: i32,
     offset: i32,
+    timestamp: i32,
 ) -> Result<Vec<String>> {
-    let res = sqlx::query(
-        "SELECT cid FROM jobs 
-                                                JOIN namespaces ON namespaces.id = jobs.ns_id 
-                                                WHERE name=$1 AND relation = $2
-                                                ORDER BY jobs.id DESC
-                                                LIMIT $3
-                                                OFFSET $4",
-    )
-    .bind(ns)
-    .bind(rel)
-    .bind(limit)
-    .bind(offset)
-    .fetch_all(pool)
-    .await?;
+    let sql = pub_cids_build_query(&ns, &rel, &limit, &offset, &timestamp);
```

Comment:
build the query for fetching cids. this query builder considers if timestamp is present or not. it is not present if it is equals `0`